captum/_utils/common.py

```diff
@@ -427,6 +436,24 @@ def _verify_select_column(
     return output[(slice(None), *target)]
 
 
+def _verify_select_neuron(
+    layer_output: Tuple[Tensor, ...],
+    selector: Union[int, Tuple[Union[int, slice], ...], Callable],
+) -> Tensor:
+    if callable(selector):
+        return selector(layer_output if len(layer_output) > 1 else layer_output[0])
+
+    assert len(layer_output) == 1, (
```

Comment:
since we are allowing slice and summing after we do this check, will this work as expected ?
I think we probably need to add some test cases when the output per example is not a scalar.